markets/perps-market/contracts/storage/GlobalPerpsMarket.sol

```diff
@@ -70,13 +75,24 @@ library GlobalPerpsMarket {
         }
     }
 
+    /**
+     * @notice Check whether, given the new locked credit delta, if the market is still solvent given its credit capacity from the core system.
+     */
+    function validateMarketCapacity(Data storage self, int256 lockedCreditDelta) internal view {
+        int256 delegatedCollateralValue = getDelegatedCollateralValue(self);
+        int256 lockedCredit = minimumCredit(self, PerpsPrice.Tolerance.DEFAULT).toInt() +
```

Comment:
I believe we'll just need to add the sUSD collateral amount to the `lockedCredit` here as this has been moved to the `PerpsMarketFactoryModule.minimumCredit function`.

Reply:
Ah I believe the getDelegatedCollateralValue should remove this amount correctly, verifying...